src/components/input/LocationAddress.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <label
+    :class="{ focused: isInputFocused }"
+    v-overlay-container="{ togglePropName: 'displaySuggestions' }"
+    v-navigable-container="{ cursorPropName: 'cursorIndex' }"
+  >
+    <input
+      type="text"
+      @input="onInputInput"
+      @focus="onInputFocus"
+      @blur="onInputBlur"
+      @keydown="onInputKeyDown"
+      v-model="query"
+      :placeholder="placeholder"
+    />
+    <ul v-if="displaySuggestions">
+      <li
+        v-for="(suggestion, index) in suggestions"
+        :key="suggestion.id"
+        @click="onSuggestionClick(index)"
+        :class="{ cursor: index === cursorIndex }"
+      >
+        <span>{{ suggestion.address }}</span>
+      </li>
+    </ul>
+  </label>
+</template>
+<style scoped lang="scss">
+@import "../../style/variables";
+
+label {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  color: $greyscale-1;
+
+  &::after {
+    content: " ";
+    display: block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 25%;
+    height: calc(100% - 2px);
+    background: linear-gradient(to right, rgba(255, 255, 255, 0), white);
+  }
+
+  &.focused::after {
+    content: none;
+  }
+}
+
+input {
+  font-size: 12px;
+  line-height: 1.83px;
+  border-width: 0 0 2px 0;
+  border-color: #4d4d4d;
+  color: #4d4d4d;
+  outline: none;
+  flex-grow: 1;
+}
+ul {
+  position: absolute;
+  top: 100%;
+  width: 100%;
+  border-width: 0;
+  border-style: solid;
+  border-color: $greyscale-1;
+  border-radius: 0 0 3px 3px;
+  background-color: white;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  box-shadow: 0 2px 4px 0 $greyscale-2;
+
+  li {
+    padding: 4px 8px;
+    font-size: 12px;
+    line-height: 1.83;
+    cursor: pointer;
+
+    &.cursor {
+      background-color: $greyscale-2;
+    }
+  }
+}
+</style>
+<script>
+import { debounce } from "lodash";
+import { mapActions } from "vuex";
+
+import "../../directives/overlayContainer";
+import "../../directives/navigableContainer";
+
+export const defaultValue = {
+  address: "",
+  coordinates: null
+};
+
+export default {
+  props: {
+    placeholder: {
+      type: String,
+      default: "Choose an address"
```

Comment:
Shouldn't this be Dutch instead of English?

Reply:
Not that I know of, website is for the non-dutch speaking population